db_scripts/mosip_idrepo/ddl/idrepo-fk.sql

```diff
@@ -26,3 +26,12 @@ ALTER TABLE idrepo.uin_biometric ADD CONSTRAINT fk_uinb_uin FOREIGN KEY (uin_ref
 REFERENCES idrepo.uin (uin_ref_id) MATCH SIMPLE
 ON DELETE NO ACTION ON UPDATE NO ACTION;
 -- ddl-end --
+
+--INDEX 
+CREATE INDEX IF NOT EXISTS idx_uin_uin_hash ON idrepo.uin USING btree(uin_hash);
```

Comment:
Why are we adding index statements to fk.sql? They should be added to their table DDL file. 
cc: @vishwa-vyom

Reply:
removed